src/main/scala/edu/ie3/simona/config/SimonaConfig.scala

```diff
@@ -6,14 +6,85 @@
 
 package edu.ie3.simona.config
 
+import com.typesafe.config.{Config, ConfigRenderOptions}
+import pureconfig._
+import pureconfig.error._
+import pureconfig.generic.ProductHint
+import pureconfig.generic.auto._
+
+import java.time.Duration
+import scala.language.implicitConversions
+import scala.util.Try
+
 final case class SimonaConfig(
     simona: SimonaConfig.Simona
-)
+) {
+  def render(options: ConfigRenderOptions): String =
+    SimonaConfig.render(this, options)
+}
+
 object SimonaConfig {
+  // pure config start
+  implicit def productHint[T]: ProductHint[T] =
+    ProductHint[T](ConfigFieldMapping(CamelCase, CamelCase))
+
+  // TODO: replace with finite duration
```

Comment:
Is this done in another PR, or shoudl we create an issue for this?

Reply:
There is an PR @marvinheintze worked on.